### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by mvafatabar**
Tada ... what do u think ?

### CODE REVIEW

**Comment by mvafatabar** [comment]
now i changed the main and navbar . hows that?

**Comment by mvafatabar** [comment]
now i added readme.md too

File: app/index.html [hunk 1]
```diff
@@ -110,15 +110,8 @@ <h5 class="opening-time__heading">Telephone</h5>
     <!--</a>-->
 
     <!-- Google Analytics: change UA-XXXXX-X to be your site's ID. -->
-    <script>
```
Comment (rayvafa) [inline app/index.html]: don't move GA tracking stuff. it's not related to what you are doing.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+
```
Comment (rayvafa) [inline README.md]: do the read me in a separate branch. don't mix up things together. it's harder to reason about everything together.

**Comment by rayvafa** [comment]
We don't need this changes anymore. I will delete the branch.

**Pull request closed without merging**